webextensions/manifest/action.json

```diff
@@ -97,9 +100,10 @@
               },
               "opera": "mirror",
               "safari": {
-                "version_added": null,
+                "version_added": false,
                 "notes": "SVG icons are not supported. Grayscale images will be treated as template icons and processed with the system accent color and system appearance."
```

Comment:
If this entire feature is `false`, then it doesn't seem like the note is needed:

```suggestion
```